sdk/ai/azure-ai-assistants/tests/model_assistants_test_base.py

```diff
@@ -0,0 +1,493 @@
+        credential = AzureKeyCredential(key)
+        api_version = kwargs.pop("azure_ai_assistant_client_api_version")
+        return endpoint, credential, api_version
+
+    # # Methods to create sync and async clients using Load_client() function
+    # async def _load_async_chat_client(self, *, bad_key: bool = False, **kwargs) -> async_sdk.AssistantsClient:
+    #     endpoint, credential = self._load_chat_credentials(bad_key=bad_key, **kwargs)
+    #     return await async_sdk.load_client(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED)
+
+    # def _load_chat_client(self, *, bad_key: bool = False, **kwargs) -> sdk.AssistantsClient:
+    #     endpoint, credential = self._load_chat_credentials(bad_key=bad_key, **kwargs)
+    #     return sdk.load_client(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED)
+
+    # async def _load_async_embeddings_client(self, *, bad_key: bool = False, **kwargs) -> async_sdk.EmbeddingsClient:
+    #     endpoint, credential = self._load_embeddings_credentials(bad_key=bad_key, **kwargs)
+    #     return await async_sdk.load_client(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED)
+
+    # def _load_embeddings_client(self, *, bad_key: bool = False, **kwargs) -> sdk.EmbeddingsClient:
+    #     endpoint, credential = self._load_embeddings_credentials(bad_key=bad_key, **kwargs)
+    #     return sdk.load_client(endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED)
+
+    # def _create_chat_client(self, *, bad_key: bool = False, **kwargs) -> sdk.AssistantsClient:
+    #     endpoint, credential = self._load_chat_credentials(bad_key=bad_key, **kwargs)
+    #     return sdk.AssistantsClient(
+    #         endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED, **kwargs
+    #     )
+
+    # # Methods to create the different sync and async clients directly
+    # def _create_async_assistant_client(self, *, bad_key: bool = False, **kwargs) -> async_sdk.AssistantsClient:
+    #     endpoint, credential = self._load_chat_credentials(bad_key=bad_key, **kwargs)
+    #     return async_sdk.AssistantsClient(
+    #         endpoint=endpoint, credential=credential, logging_enable=LOGGING_ENABLED, **kwargs
+    #     )
+
+    # def _create_assistant_client(
+    #     self, *, key_auth: bool = True, bad_key: bool = False, **kwargs
+    # ) -> sdk.AssistantsClient:
+    #     endpoint, credential, credential_scopes, headers, api_version = self._load_aoai_chat_credentials(
+    #         key_auth=key_auth, bad_key=bad_key, **kwargs
+    #     )
+    #     return sdk.AssistantsClient(
+    #         endpoint=endpoint,
+    #         credential=credential,
+    #         credential_scopes=credential_scopes,
+    #         headers=headers,
+    #         api_version=api_version,
+    #         logging_enable=LOGGING_ENABLED,
+    #     )
+
+    def _create_async_aoai_chat_client(

```

Comment:
I would prefer not to have _create_async_aoai_chat_client helper but use AssistantsClient directly in test case